recipes-core/images/README

```diff
@@ -13,7 +13,7 @@ c     | * nilrt-runmode-initramfs.bb
 cn xpl| * nilrt-runmode-rootfs.bb
       | * nilrt-runmode-system-image.bb
 c   pl| * nilrt-safemode-initramfs.bb
-      | * nilrt-safemode-image.bb        # NILRT safemode (firmware) rootfs
```

Comment:
I'm confused by this renaming. How is nilrt-safemode-image different from nilrt-runmode-system-image? They are both images, right? I would like to keep the image part of a name to indicate that's it's the top-level recipe I need to be building.

Reply:
The `nilrt-safemode-image` (now `nilrt-safemode-rootfs`) is the safemode initramfs (`ramdisk.xz` on disk?), plus the non-volatile rootfs files which are shared between safemode and runmode (eg. the /etc/ files which are shared between the mounts.)

`nilrt-runmode-system-image` is an archive of the `nilrt-runmode-rootfs`, plus the runmode post-installation script and CDF metadata necessary to make a single payload that MAX can install.

>I would like to keep the image part of a name to indicate that's it's the top-level recipe I need to be building.

I considered that, but preferred to make it clear what kind of image it was instead. And I'm not sure its meaningful to say that some of the images are "top-level" versus other ones. The `nilrt-safemode-rootfs` is what is used to reformat the "firmware" image through MAX, but it is only a payload in the context of provisioning - where the top level recipe is `nilrt-recovery-image`. Likewise, both `nilrt-runmode-rootfs` and `nilrt-runmode-system-image` are viable standalone packages, just for different purposes.

Reply:
What I meant by "top-level" is that both of those packages - `nilrt-safemode-image` and `nilrt-runmode-system-image` can be used as is to install a functional safemode and runmode on a target. I understand that the `nilrt-recovery-image` is an additional means to install the safemode, but because the `nilrt-safemode-image` can also be used to install a complete image, I would prefer to keep the `-image` part of the name.

Reply:
Let's talk offline at the knowledge sharing today.

Reply:
We reviewed this decision as a team and decided not to add the `-image` suffix.